src/main/java/seedu/algobase/logic/commands/AddTagCommand.java

```diff
@@ -0,0 +1,52 @@
+package seedu.algobase.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.algobase.logic.parser.CliSyntax.PREFIX_TAG;
+
+import seedu.algobase.logic.commands.exceptions.CommandException;
+import seedu.algobase.model.Model;
+import seedu.algobase.model.tag.Tag;
+
+/**
+ * Adds a Tag to the algobase.
+ */
+public class AddTagCommand extends Command {
+
+    public static final String COMMAND_WORD = "newtag";
```

Comment:
Can we have `addtag` so that it's consistent with `add` and `addplan`?

Reply:
I've followed the UG and I also think `addtag` is better.
I think we can change the command in UG.

Reply:
Sure, go ahead and change the UG.